babel_util/luigi/luigi_aminer_test.py

```diff
@@ -0,0 +1,27 @@
+# In order to run with luigi command, this folder must be added to sys.path
```

Comment:
What exactly does this do? Is it testing the task? If so, it should be in a tests/ folder in this directory.